components/automate-ui/src/app/pages/infra-proxy/chef-servers-details/chef-servers-details.component.html

```diff
@@ -0,0 +1,133 @@
+              Description is required.
+            </chef-error>
+          </chef-form-field>
+          <chef-form-field id="update-fqdn">
+            <label>
+              <span class="label">FQDN <span aria-hidden="true">*</span></span>
+              <input chefInput formControlName="fqdn" type="text" autocomplete="off"
+                data-cy="update-chefServer-fqdn">
+            </label>
+            <chef-error
+              *ngIf="(updateServerForm.get('fqdn').hasError('required') || updateServerForm.get('fqdn').hasError('pattern')) && updateServerForm.get('fqdn').dirty">
+              FQDN is required.
+            </chef-error>
+          </chef-form-field>
+          <chef-form-field id="update-ip_address">    
+            <label>
+              <span class="label">IP Address <span aria-hidden="true">*</span></span>
+              <input chefInput formControlName="ip_address" type="text" autocomplete="off"
+                data-cy="update-chefServer-ip_address">
+            </label>
+            <chef-error
+              *ngIf="(updateServerForm.get('ip_address').hasError('required') || updateServerForm.get('ip_address').hasError('pattern')) && updateServerForm.get('ip_address').dirty">
+              FQDN is required.
+            </chef-error>
+          </chef-form-field>  
+          <div id="button-bar">
+            <chef-button [disabled]="isLoading || !updateServerForm.valid || !updateServerForm.dirty"
+              primary inline (click)="saveServer()">                
+            <chef-loading-spinner *ngIf="saving"></chef-loading-spinner>
+            <span *ngIf="saving">Saving...</span>
+            <span *ngIf="!saving">Save</span></chef-button>
+            <span id="saved-note" *ngIf="saveSuccessful && !updateServerForm.dirty">All changes saved.</span>
+          </div>
+        </form>
+      </section>
+      <section class="page-body" *ngIf="tabValue === 'orgs'">
+        <ng-container>
+          <chef-toolbar>
+            <chef-button id="create-button" primary (click)="openCreateModal('create')">Create Org</chef-button>
+          </chef-toolbar>
+          <chef-table-new>
+            <chef-table-header>
+              <chef-table-row>
+                <chef-table-header-cell>Org Name</chef-table-header-cell>
+                <chef-table-header-cell>Admin</chef-table-header-cell>
+                <chef-table-header-cell class="controls"></chef-table-header-cell>
+              </chef-table-row>
+            </chef-table-header>
+            <chef-table-body>
+              <chef-table-row *ngFor="let org of orgs">
+                <chef-table-cell>
+                  <a [routerLink]="['/infrastructure','chef-servers', server?.id, 'orgs', org.id, 'cookbooks']">{{ org.name }}</a>
+                </chef-table-cell>
+                <chef-table-cell>{{ org.admin_user }}</chef-table-cell>
+                <chef-table-cell class="controls">
```

Comment:
Another recent change was to standardize on "three-dot-column" for this class wrapping the control menu.
```suggestion
                <chef-table-cell class="three-dot-column">
```